src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -6,8 +6,8 @@ export class FilterOption extends HTMLElement {
     super();
 
     this.viewCheckboxLabels = new Set();
-    // It contains the checked checkboxes
-    this.activeCheckboxLabels = new Set();
+    // It contains the unchecked checkboxes
+    this.inActiveCheckboxLabels = new Set();
```

Comment:
Rename to `inactive`, `inActiveCheckboxLabels` looks like "in active checkbox labels", which doesn't reflect the actual purpose.

Reply:
done :-)